packages/flex-plugin-webpack/src/devServer/pluginServer.ts

```diff
@@ -171,6 +171,36 @@ export const _makeRequestToFlex = async (token: string, path: string, version?:
   });
 };
 
+const splitTokenToChunks = (token: string, length: number): string[] => {
+  if (length <= 0) {
+    throw new Error('Length must be a positive integer');
+  }
+
+  const tokenChunks: string[] = [];
+  for (let i = 0; i < token.length; i += length) {
+    const chunk = token.slice(i, i + length);
+    tokenChunks.push(chunk);
+  }
+
+  return tokenChunks;
+};
+
+const combineJweToken = (cookies: { [key: string]: string }): string => {
+  const chunks: string[] = [];
+  let key = 0;
+
+  while (true) {
+    const token = cookies[`flex-jwe${key === 0 ? '' : `-${key + 1}`}`];
```

Comment:
I believe using key++ and initialising key = 1 is more readable than manually incrementing on line 198.